### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by ghost** [MC-PE/Translator#2]
[Server thread/CRITICAL]: Could not pass event 'pocketmine\event\server\DataPacketSendEvent' to 'Translator v1': Call to a member function getName() on null on Ad5001\Translator\Main
[14:04:09] [Server thread/CRITICAL]: Error: "Call to a member function getName() on null" (EXCEPTION) in "/Translator/src/Ad5001/Translator/Main" at line 69

Please, fix it. It is a great plugin.

**Comment by Ad5001** [comment]
Unfortunatly, soon, bing will shutdown their translation service and we cannot offer any other one (1 million charcters on google = $20). This plugin will be shut down.

**Comment by ghost** [comment]
Please :/. And if will someone pay $20? Is it 
one-time payment?

**Comment by Ad5001** [comment]
No, it's per mouth and per million of characters (which isn't much thought the old free limit was 2 millions and it got passed it in 3 weeks).
But the problems are also that there is no longer any free translator API available on the Internet (except REALLY crappy ones that are only out for a few languages).

**Comment by ghost** [comment]
And is there any different Language Translator or changer plugin for MC:PE does not need internet connection?

**Comment by Ad5001** [comment]
Don't know to be honest if a such open source developer would create a local translator...

**Comment by ghost** [comment]
I found YourLang from BoxOfDevs team but when I type a command in game, it says - [Error] This command must be use in-game
